uefi-raw/src/table/boot.rs

```diff
@@ -363,6 +364,25 @@ pub struct MemoryDescriptor {
 impl MemoryDescriptor {
     /// Memory descriptor version number.
     pub const VERSION: u32 = 1;
+
+    /// Returns a best-effort **guess** of the `desc_size` that UEFI will
+    /// report for a [`MemoryDescriptor`]. You may use this in unit testing. In
+    /// integration tests and real world scenarios, refrain from using this, and
+    /// use the actual reported `desc_size` reported from `UEFI#GetMemoryMap()`
+    /// instead.
+    ///
+    /// # Safety
+    /// Unsafe because you should always use the `desc_size` that is associated
+    /// with the memory map.
+    #[must_use]
+    pub const unsafe fn uefi_desc_size() -> usize {
```

Comment:
idea here is to use it in unit tests or so. Not used anywhere now, however.

Reply:
I think we should drop this, there's no circumstance where code should need to guess at the underlying size of descriptors, even in tests; `GetMemoryMap` must always be used.

Reply:
okay, dropped.